packages/editor/src/store/actions.js

```diff
@@ -815,8 +815,8 @@ export function updateSharedBlockTitle( id, title ) {
 	return updateReusableBlockTitle( id, title );
 }
 
-export function convertBlockToSaved( clientId ) {
-	deprecated( 'convertBlockToSaved', {
+export function convertBlockToShared( clientId ) {
```

Comment:
Should we extend depreciation period for this one to ensure plugins can notice it?

Reply:
I'm not really sure what to do in this situation. We've already shipped the misnamed function which means that we already "broke" third party code in 3.4. I don't see any issues about this so it's likely that nobody was using this selector. I'm tempted to just get rid of it all in 3.6 as planned.

Reply:
yes, the chances are really small, let's just get rid of it.